### DISCUSSION TITLE
Richiesta nuova

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#132] [Idee e proposte]
Buongiorno,sarebbe possibile implementare già durante la compilazione del registro la data di scadenza dei compiti senza andare sia nell'agenda che scriverlo su attività?

**Comment by trinko** [comment]
Attualmente no. 
Comunque consiglio l'uso della sola agenda: in questo modo, nella pagina iniziale viene ricordato all'alunno/genitore la presenza di compiti sia il giorno precedente che quello della scadenza.
